models/hospital.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+const { Model } = require("sequelize");
+module.exports = (sequelize, DataTypes) => {
+  class Hospital extends Model {}
+  Hospital.init(
+    {
+      id: {
+        field: "hospital_uuid",
+        type: DataTypes.UUID,
+        primaryKey: true,
+        autoIncrement: true,
+      },
+      hospitalName: {
+        field: "hospitalname",
+        type: DataTypes.STRING,
+        unique: true,
+        allowNull: false,
+      },
+      recordUpdateSource: {
+        field: "recordupdatesource",
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
+      recordUpdateTimestamp: {
```

Comment:
We'll want to manually configure the timestamps just like all the other attributes...